eclipse/plugin.xml

```diff
@@ -1002,6 +1002,7 @@
       <menuContribution
             allPopups="false"
             locationURI="popup:org.eclipse.ui.popup.any?before=menustart">
+         <!-- See SarosView, the context menu is created manually 
```

Comment:
Why are you using a comment instead of simply removing the stuff and adding the comment?

Reply:
Well I can removed it. I just did not removed it in case of E4 migration but I guess this will never happen.

I tried to find some guides (especially mixing E3 and E4 stuff) and I failed badly. Most guides are for RCP applications and I do not know if this also applies to plugins.

Reply:
Yes, I have a similar experience. I also tried [this](https://jaxenter.de/tutorial-so-portieren-sie-eclipse-3-anwendungen-auf-eclipse-4-39492) german guide. I was able to start the SarosView as a e4view, but the Toolbar was still missing. I am still thinking about the relation of `application.e4xmi`, `fragment.e4xmi` and the `plugin.xml`. I will try it again later.